test/user.test.js

```diff
@@ -67,6 +68,26 @@ describe('User', function() {
       });
     });
 
+   it('Create a new user (email case-sensitivity off)', function(done) {
+      User.settings.caseSensitiveEmail = false;
+      User.create({email: 'F@b.com', password: 'bar'}, function(err, user) {
+        assert(!err);
```

Comment:
Please use this form: `if (err) return done(err)`. It produces stack traces pointing to the original error, as opposed to `assert(!err)` which points to this line in tests. Please apply this rule to all new tests you are adding in this patch (no need to change existing code).